packages/protocol/contracts/common/FeeHandler.sol

```diff
@@ -48,19 +50,29 @@ contract FeeHandler is
     uint256 toDistribute;
     // Historical amounts burned by this contract
     uint256 pastBurn;
+    uint256 lastLimitDay;
+  }
+
+  struct Beneficiary {
+    FixidityLib.Fraction fraction;
+    string name;
+    bool exists;
   }
 
-  uint256 public constant FIXED1_UINT = 1000000000000000000000000; // TODO move to FIX and add check
+  uint256 public constant FIXED1_UINT = 1000000000000000000000000;
 
   // Min units that can be burned
   uint256 public constant MIN_BURN = 200;
 
   // last day the daily limits were updated
-  uint256 public lastLimitDay;
+  uint256 private deprecated_lastLimitDay; // deprecated
 
-  FixidityLib.Fraction public burnFraction; // 80%
+  // TODO try to make this private if it doesn't break storage
+  // reason it's inverse it's because it used to be burnFraction and was migrated
+  // ignoreRenaming_ prefix allows the tooling to ignore the variable renaming
+  FixidityLib.Fraction private ignoreRenaming_inverseCarbonFraction; // 80%
```

Comment:
I don't think the storage checker handles `ignoreRenaming_`, only `deprecated_`?